compiler/testData/codegen/box/reflection/properties/propertyName.kt

```diff
@@ -0,0 +1,12 @@
+class A {
+    val a = ""
+    val b = ""
+}
+
+fun box(): String {
+    val a = A::a.name
+    if (a != "a") return "Fail $a"
+    val b = A::`b`.name
+    if (b != "b") return "Fail $b"
+    return "OK"
```

Comment:
Should I hook into `ExpressionCodegen#visitSimpleNameExpression` and prevent the anonymous class from being generated or is there an approach where I can remove the generated class afterwards?